yarn.lock

```diff
@@ -10705,7 +10693,7 @@ node-sass-import-once@^1.2.0:
   dependencies:
     js-yaml "^3.2.7"
 
-node-sass@^4.6.0, node-sass@^4.7.2:
+node-sass@^4.6.0:
```

Comment:
Bump `@addepar/style-toolbox` to `0.8.1` and this should also go away

Reply:
we haven't published `0.8.1` yet